cookbooks/hdfsdu/metadata.rb

```diff
@@ -0,0 +1,11 @@
+name             'hdfsdu'
+maintainer       'Bloomberg Finance L.P.'
+license          'All rights reserved'
```

Comment:
The `license` should be [`Apache License 2.0`](https://github.com/bloomberg/chef-bach/blob/89e3a11f153f6652fdf07e7a0d1d532a9b4ecd3e/cookbooks/bcpc/metadata.rb#L4)

Reply:
Updated